base/ca/shared/profiles/ca/estFullcmcUserCert.cfg

```diff
@@ -0,0 +1,86 @@
+profileId=estFullCMCUserCert
+auth.instance_id=CMCAuthForEST
+authz.acl=group="Certificate Manager Agents"
+desc=This certificate profile is for enrolling user certificates by using the agent-signed CMC certificate request via EST fullcmc.
+enable=true
+enableBy=caadmin
+input.i1.class_id=cmcCertReqInputImpl
+input.list=i1
+name=Agent-Signed CMC-Authenticated User Certificate Enrollment via EST fullcmc
+output.list=o1
+output.o1.class_id=certOutputImpl
+policyset.cmcUserCertSet.1.constraint.class_id=subjectNameConstraintImpl
+policyset.cmcUserCertSet.1.constraint.name=Subject Name Constraint
```

Comment:
In [estServiceCert.cfg](https://github.com/dogtagpki/pki/blob/master/base/ca/shared/profiles/ca/estServiceCert.cfg) there is a constraint (number 20) to verify and limit the subject of the new certificate (using EST information in input i3) but with CMC I do not find constraints. Are these not needed?